sup3r/cli.py

```diff
@@ -116,16 +118,16 @@ def forward_pass(ctx, verbose):
             "execution_control": {
                 "option": "local"
             },
-            "execution_control_eagle": {
-                "option": "eagle",
+            "execution_control_kestrel": {
```

Comment:
Not a GAPs thing. All args should go in the `execution_control` block for sure. I kept it around since you guys used it in your documentation for eagle args. Do you want me to replace the original `execution_control` block with the HPC required args? Or just delete it entrierly?

Reply:
Yeah I'm not sure why we had that lol. The former would be great, thanks.